docker-compose.yml

```diff
@@ -21,7 +21,10 @@
     "links": [
       "redis:localhost"
     ],
-    "privileged": true
+    "privileged": true,
+    "ports": [
+      "5554:5432" 
```

Comment:
Why the need to expose this?

Reply:
For me it's very convenient to load tables in QGIS

Reply:
That's cheating :P